docs/tutorials/ad-hoc-simulation/planning-simulation.md

```diff
@@ -2,81 +2,130 @@
 
-   ```bash
-   source ~/autoware/install/setup.bash
-   ros2 topic pub /autoware/engage autoware_auto_vehicle_msgs/msg/Engage "engage: true" -1
-   ```
+b) In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the initial pose. An image representing the vehicle should now be displayed.
 
-   ![start-driving](images/planning/lane-following/start-driving.png)
+!!! warning
 
-6. To delete objects, click the `Delete All Objects` button in the toolbar.
+    Remember to set the initial pose of the car in the same direction as the lane.
 
-   ![delete-objects](images/planning/lane-following/delete-objects.png)
+    To confirm the direction of the lane, check the arrowheads displayed on the map.
 
-7. After that, the vehicle will restart driving and reach the goal.
+#### 4. Set a goal pose for the ego vehicle
 
-   ![restart-driving](images/planning/lane-following/restart-driving.png)
+a) Click the `2D Goal Pose` button in the toolbar, or hit the `G` key.
 
-   ![reach-goal](images/planning/lane-following/reach-goal.png)
+b) In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the goal pose. If done correctly, you will see a planned path from initial pose to goal pose.
 
-## How to simulate parking maneuvers
+![set-goal-pose](images/planning/lane-following/set-goal-pose.png)
+
+#### 5. Engage the ego vehicle
+
+Now you can start the ego vehicle driving by clicking the `Engage` button in `AutowareStatePanel`. Alteratively, you can manually engage the vehicle by running the following command:
+
+```bash
+source ~/autoware/install/setup.bash
+ros2 topic pub /autoware/engage autoware_auto_vehicle_msgs/msg/Engage "engage: true" -1
+```
+
+![start-driving](images/planning/lane-following/engage-and-start-planning.png)
+
+### Parking scenario
 
 1. Set an initial pose and a goal pose, and engage the ego vehicle.
 
    ![after-set-goal-pose](images/planning/parking/after-set-goal-pose.png)
 
 2. When the vehicle approaches the goal, it will change to the parking mode.
+3. After that, the vehicle will start driving backwards and park at the destination parking spot.
 
-   ![change-to-parking-mode](images/planning/parking/change-to-parking-mode.png)
+   ![parking-maneuver](images/planning/parking/parking-maneuver.png)
 
-3. After that, the vehicle will start driving backwards and park at the destination parking spot.
+## Advanced Simulations
+
+### Placing dummy objects
+
+1. Click the `2D Dummy Car` or `2D Dummy Pedestrian` button in the toolbar.
+
+2. Set the pose of the dummy object by clicking and dragging on the map.
+
+3. Set the velocity of the object in `Tool Properties -> 2D Dummy Car/Pedestrian` panel.
+
+![set-dummy-car](images/planning/lane-following/place-dummy-car.png) 4. Delete any dummy objects placed in the view by clicking the `Delete All Objects` button in the toolbar.
+
+### Traffic light recognition simulation
+
```

Comment:
My understanding is that traffic lights on the map are treated as green by default, so added some introductory text to explain that point. Please amend as necessary if my understanding is incorrect.

```suggestion
By default, traffic lights on the map are all treated as if they are set to green. As a result, when a path is created that passed through an intersection with a traffic light, the ego vehicle will drive through the intersection without stopping. 

The following steps explain how to set and reset traffic lights in order to test how the Planning component will respond.

```

Reply:
Yes that's right. This looks more clear.